gradle/ext.gradle

```diff
@@ -3,36 +3,36 @@ println "tasks: $gradle.startParameter.taskNames"
 // set some global variables
 gradle.ext {
     kotlin_version = kotlin_version
-    kotlinx_coroutines_version = '1.4.3-native-mt'
+    kotlinx_coroutines_version = '1.5.0-native-mt'
```

Comment:
why not 1.5.1?

Reply:
1.5.1 requires kotlin 1.5.20 which is causes problems with compose plugin